### DISCUSSION TITLE
Sending emails via distribution list?

### DISCUSSION THREAD

**Discussion by yegorsb** [O365/python-o365#663] [Q&A]
I was wondering if there was a way to send an email via email that is in a distribution list/group?
Here is an example that I am trying:
```
credentials = ('xxxxxxxxx','xxxxxxxx')
account = Account(credentials,auth_flow_type = 'credentials', tenant_id = 'xxxxxxx-xxxx-xxxx-xxxx-xxxxxx')

if account.authenticate():
    print('Authenticated!')

mailbox = account.mailbox('xxx@xxx.org')
m = mailbox.new_message()
m.to.add('xxxxx@xxx.org')
m.body = 'test'
m.subject = 'test'
m.send()
```

This code does work if the sender email address (mailbox) is tied to a user... but how about group that is tied to a distribution list? We have an email associated to a distribution group but get a user invalid error. I was wondering if there was a way to do this or if there isn't.

Thanks!

**Comment by alejcas** [comment]
I don't know if it's possible.

Check the [Ms Graph docs](https://docs.microsoft.com/en-us/graph/api/overview?view=graph-rest-1.0) to see if it's possible there. 
If it's possible then it can be implemented into this library.
